### ISSUE TITLE
[Tarea 2] P2 - Probabilidad

### ISSUE THREAD

**Issue by VicenteMerino** [marceloarenassaavedra/IIC2283-2-21#45]
Hola, en esta pregunta podemos jugarnosla con una probabilidad menor para bajar el tiempo y si da bien está ok?

**Comment by marceloarenassaavedra** [comment]
Sí, puedes hacer esto. Si en todos los test tu resultado es correcto, entonces vas a tener un 7. Pero por supuesto existe una mayor probabilidad de que tu algoritmo se equivoque :)

Saludos!
